packages/react-native-aztec/android/build.gradle

```diff
@@ -116,34 +123,27 @@ dependencies {
     }
 }
 
-if (willPublishReactNativeAztecBinary) {
-    project.afterEvaluate {
-        publishing {
-            publications {
-                ReactNativeAztecPublication(MavenPublication) {
-                    artifact bundleReleaseAar
+project.afterEvaluate {
+    publishing {
+        publications {
+            ReactNativeAztecPublication(MavenPublication) {
+                artifact bundleReleaseAar
 
-                    groupId 'org.wordpress-mobile.gutenberg-mobile'
-                    artifactId 'react-native-aztec'
-                    version publishReactNativeAztecVersion
+                groupId 'org.wordpress-mobile.gutenberg-mobile'
+                artifactId 'react-native-aztec'
+                // version is set by 'publish-to-s3' plugin
 
-                    addDependenciesToPom(pom)
-                }
-            }
-            repositories {
-                maven {
-                    url "s3://a8c-libs.s3.amazonaws.com/android"
-                    credentials(AwsCredentials) {
-                        accessKey project.hasProperty('awsAccessKey') ? project.properties['awsAccessKey'] : System.getenv('AWS_ACCESS_KEY')
-                        secretKey project.hasProperty('awsSecretKey') ? project.properties['awsSecretKey'] : System.getenv('AWS_SECRET_KEY')
-                    }
-                }
+                addDependenciesToPom(pom)
             }
         }
-    }
+   }
+}
+
+publishToS3Plugin {
+    mavenPublishGroupId 'org.wordpress-mobile.gutenberg-mobile'
+    mavenPublishArtifactId 'react-native-aztec'
```

Comment:
These properties are used to check if a version is already published to S3. It should match the information in the `publish` block.

I am going to re-visit this later and try to get rid of it - or at least extract the values so we don't have to reference them twice.